libpod/runtime_pod_infra_linux.go

```diff
@@ -30,8 +30,8 @@ func (r *Runtime) makeInfraContainer(ctx context.Context, p *Pod, imgName, imgID
 		return nil, err
 	}
 
-	// Set Pod hostname as Pod name
-	g.Config.Hostname = p.config.Name
+	// Set Pod hostname
+	g.Config.Hostname = p.config.Hostname
```

Comment:
If pod hostname isn't configured, what does the hostname become? If it ends up empty (or the name of the infra container) we may wanna fallback on pod name

Reply:
Nevermind you did that later lol